dbqueries.py

```diff
@@ -114,11 +122,59 @@ class QueryDict(TypedDict):
             }
         }
     },
+    'udw_duplicate_course_ids': {
+        'output_file_name': 'udw_duplicate_course_ids.csv',
+        'data_source': 'UDW',
+        'query_name': 'UDW Duplicate Course IDs',
```

Comment:
Well since it's specific to 120 days, it would be "UDW Duplicate Course IDs in last 120 days"

Reply:
It's actually 120 days from the start of any term that was before the current time. Not super easy to grok I know! 🤣

Reply:
I'm open to simplification here, or a change in the period. I just didn't want to rely on or modify the date end of these terms since they are often but not always 10 years out. Maybe they are more consistent now, I could check

Reply:
What do you all think of "in Active Term(s)"? That seems short while also being accurate?

Reply:
Seems good to me. I believe only the start date can be relied on to be consistent, which looks to be the true semester start date. 

Not all terms get bumped out 10 years and I'm not sure if this is even needed anymore. I've felt like the other dates (Teachers can access, TAs can access, Designers can access) could be filled in to provide the better functionality. But we work with what we've got!